Dockerfile.s390x

```diff
@@ -133,7 +133,7 @@ RUN useradd --create-home --gid docker unprivilegeduser
 
 VOLUME /var/lib/docker
 WORKDIR /go/src/github.com/docker/docker
-ENV DOCKER_BUILDTAGS apparmor selinux
+ENV DOCKER_BUILDTAGS apparmor pkcs11 selinux
```

Comment:
@tophj-ibm good point, since we aren't building notary on gccgo I'll remove pkcs11.

As @endophage mentioned above we're only using pkcs11 on the CLI-side, so just keep me posted with whether we need it in this Dockerfile and I'll update accordingly :)